drivers/analog/dac.c

```diff
@@ -224,16 +222,6 @@ static int dac_close(FAR struct file *filep)
   return ret;
 }
 
-/****************************************************************************
- * Name: dac_read
- ****************************************************************************/
-
-static ssize_t dac_read(FAR struct file *filep, FAR char *buffer,
-                        size_t buflen)
-{
-  return 0;
-}
-
```

Comment:
Good finding! I think a users will not read from DAC device! :-)

Reply:
The driver is registered with `register_driver(path, &dac_fops, 0222, dev);` -- write only